components/profile/payment-item.tsx

```diff
@@ -0,0 +1,57 @@
+import { useRouter } from "next/router";
+
+import Button from "components/button";
+import { FlexColumn, FlexRow } from "components/profile/wallet-balance";
+
+import { formatNumberToCurrency } from "helpers/formatNumber";
+
+import { Payment } from "interfaces/payments";
+
+import useNetworkTheme from "x-hooks/use-network";
+
+export default function PaymentItem({ ammount, issue, transactionHash, id, labelBounty, labelToken } : Payment) {
+  const { push } = useRouter();
+
+  const { getURLWithNetwork } = useNetworkTheme();
+
+  const CONTAINER_CLASSES = [
+    "justify-content-between align-items-center bg-transparent",
+    "border border-dark-gray border-radius-8 mb-2 py-3 px-4"
+  ];
+
+  function handleClick() {
```

Comment:
handle-click should be handled by the parent component, this should be a component argument